src/encoder.rs

```diff
@@ -383,16 +396,28 @@ impl Sequence {
       second_forward_idx >= 0
     }
   }
+
+  #[inline(always)]
+  pub fn sb_size_log2(&self) -> usize {
```

Comment:
really wish this could be a const fn augh

Reply:
It cannot be `const fn`, since it should depend on a (non-existing yet) command-line parameter (64×64 vs 128×128).

We could store them as `FrameInvariants` fields though:

```
    pub sb_size: usize,
    pub sb_size_log2: usize,
```